8.5-java11/Dockerfile

```diff
@@ -31,6 +31,17 @@ COPY tmp/shared/app_insights/ApplicationInsights.xml /usr/local/app_insights/tom
 
 RUN apk add --update openssh-server bash openrc \
         && rm -rf /var/cache/apk/* \
+        # Remove unnecessary services
+        && rm -f /etc/init.d/hwdrivers \
+                 /etc/init.d/hwclock \
+                 /etc/init.d/hwdrivers \
```

Comment:
>/etc/init.d/hwdrivers \ [](start = 17, length = 23)

This is added twice.

Reply:
fixed locally.

---
In reply to: [264959126](https://github.com/Azure-App-Service/tomcat/pull/29#discussion_r264959126) [](ancestors = 264959126)